lib/vicadmin/validate.go

```diff
@@ -180,7 +180,7 @@ func NewValidator(ctx context.Context, vch *config.VirtualContainerHostConfigSpe
 	nwErrors := []error{}
 
 	for _, host := range hosts {
-		conn, err := net.Dial("tcp", host)
+		conn, err := net.DialTimeout("tcp", host, time.Minute/2)
```

Comment:
Heck, Go might even be smart enough to realize that operation is a constant and do it at compile-time, idk, I don't think it's an important optimization lmao